src/components/PostList/PostList.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Post } from '../Post';
+
+export const PostList = ({ preparedListOfPosts }) => (
+  preparedListOfPosts.map(
+    item => (<Post {...item} key={item.id} />),
+  )
+);
+
+PostList.propTypes = {
+  preparedListOfPosts: PropTypes.arrayOf(
+    PropTypes.shape().isRequired,
```

Comment:
add post shape here, don't leave it empty

Reply:
Done